src/main/topLevelPages/myDashboard/ExploreSpaces/JourneyTile.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Visual } from '../../../../domain/common/visual/Visual';
+import { Avatar, Box, Paper, Skeleton } from '@mui/material';
+import RouterLink from '../../../../core/ui/link/RouterLink';
+import GridItem from '../../../../core/ui/grid/GridItem';
+import withElevationOnHover from '../../../../domain/shared/components/withElevationOnHover';
+import { gutters } from '../../../../core/ui/grid/utils';
+import { JourneyTypeName } from '../../../../domain/journey/JourneyTypeName';
+import { alpha } from '@mui/material/styles';
+import webkitLineClamp from '../../../../core/ui/utils/webkitLineClamp';
+import { BlockTitle } from '../../../../core/ui/typography';
+import InsertPhotoOutlinedIcon from '@mui/icons-material/InsertPhotoOutlined';
+import defaultJourneyCardBanner from '../../../../domain/journey/defaultVisuals/Card.jpg';
```

Comment:
@ccanos this is not related to the PR but it's something that I would like to suggest as a technical improvement. It's already a standard to use @app and do absolute imports instead of these deeply nested ones. Bot's suggestion is even better.
Let me know your thoughts and I'll extract a technical task on this one.